.circleci/config.yml

```
@@ -1 +1,10 @@
 version: 2.1
+executors:
+  ci-executor:
+    docker:
+      - image: REPLACE-ME-WITH-DOCKER-IMAGE
+jobs:
+  build:
+    executor: ci-executor
+    steps:
```

Good job with the multi line comment
```suggestion
jobs:
  build:
    executor: ci-executor
    steps:
```